draft-ietf-add-ddr.md

```diff
@@ -451,12 +451,20 @@ authentication-domain-name (ADN).
 # Security Considerations
 
 Since clients can receive DNS SVCB answers over unencrypted DNS, on-path
-attackers can prevent successful discovery by dropping SVCB queries or answers.
+attackers can prevent successful discovery by dropping SVCB queries or answers,
+and thus prevent clients from switching to use encrypted DNS.
 Clients should be aware that it might not be possible to distinguish between
 resolvers that do not have any Designated Resolver and such an active attack.
 To limit the impact of discovery queries being dropped either maliciously or
 unintentionally, clients can re-send their SVCB queries periodically.
 
+{{Section 8.2 of !I-D.ietf-add-svcb-dns}} describes a second downgrade attack
+where an attacker can block connections to the encrypted DNS server,
+and recommends that clients prevent it by switching to SVCB-reliant behavior once
+SVCB resolution does succeed. For DDR, this means that once a client attempts
```

Comment:
I think "once a client discovers a designation" is more accurate than "once a client attempts an encrypted DNS connection" because we draw that line elsewhere in the text, with a MUST no less. It's pedantic when the client always uses designations, but I think the distinction is important.

Reply:
SVCB-DNS uses the words "SVCB resolution succeeds" to describe this condition.  Note that "succeeds" is being used in a highly specific sense here, corresponding to [Step 4 of Section 3 of SVCB](https://mikebishop.github.io/dns-alt-svc/draft-ietf-dnsop-svcb-https.html#name-client-behavior): "one or more "compatible" ServiceMode records are returned".

If you prefer the term "designation", it would be equivalent to say "once the client discovers a _compatible_designation".

Reply:
I think @mstojens 's comment was about the sentence _after_ "SVCB resolution does succeed". We can keep that part. However, we can talk about a compatible or supported designation in the latter sentence.